src/database/queries.rs

```diff
@@ -71,3 +73,15 @@ where
         .filter(transactions::id.eq(tx_id))
         .first::<Transaction>(&conn)
 }
+
+pub async fn delete_txs<Context>(ctx: &Context, current_epoch: u128, epoch_amount: u128) -> Result<usize, Error>
+where
+    Context: QueryContext,
+{
+    let epochs : Vec<Epoch> = (0..epoch_amount).map(|i| Epoch(current_epoch - i)).collect();
+    let conn = ctx.get_db_connection();
+    let txs = transactions
```

Comment:
I would call this `tx_filter` or just `filter`. When I was reading the code, my first impression was that you queried all transactions that quite were about to delete, which isn't true, but that's the first impression I had when reading the code.